tests/devices/test_default_qubit.py

```diff
@@ -547,11 +562,12 @@ def test_apply_operation_single_wire_with_parameters(
         operations that have parameters."""
 
         # parameter = par[0]
```

Comment:
Out of scope, just stumbled upon it :sweat_smile: 
```suggestion
```

Reply:
Done